charts/rbac/templates/devops-jr/argocd.yaml

```diff
@@ -0,0 +1,28 @@
+{{ if .Values.rbac.devopsJr.argocd }}
+
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+ name: saritasa-devops-jr-role
+ namespace: argo-cd
+rules:
+  - apiGroups: ["argoproj.io"]
+    resources: ["applications", "appprojects"]
+    verbs: ["create", "get", "list", "watch", "update", "patch"]
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: saritasa:devops-jr
+  namespace: argo-cd
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: saritasa-devops-jr-role
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: saritasa:sso:devops-jr
+
+{{ end }}
```

Comment:
No let's remove devopsJR from the chart. The REAMDE has this

```
You can also add any ad-hoc RBAC role/bindings via .Values.extraRbac multiline string

For example:

```yaml
extraRbac: |
  kind: Role
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
   name: cheesecake-users-readonly-role
   namespace: {{ .Release.Namespace }}
  rules:
    - apiGroups: [""]
      resources:  ["services", "pods", "pods/log", "configmaps"]
      verbs: ["get", "list", "watch"]
    - apiGroups: [""]
      resources:  ["pods"]
      verbs: ["delete"]
    - apiGroups: [""]
      resources: ["pods/portforward", "services/portforward", "pods/exec"]
      verbs: ["create"]
    - apiGroups: ["extensions"]
      resources:  ["*"]
      verbs: ["get", "list", "watch"]
    - apiGroups: ["batch"]
      resources:  ["*"]
      verbs: ["get", "list", "watch"]
    - apiGroups: ["apps"]
      resources:  ["*"]
      verbs: ["get", "list", "watch"]
    - apiGroups: ["tekton.dev"]
      resources:  ["*"]
      verbs:  ["taskruns", "pipelineruns", "pipelines", "tasks"]

  ---
  kind: RoleBinding
  apiVersion: rbac.authorization.k8s.io/v1
  metadata:
    name: cheesecake:users
    namespace: {{ .Release.Namespace }}
  roleRef:
    apiGroup: rbac.authorization.k8s.io
    kind: Role
    name: cheesecake-users-readonly-role
  subjects:
  - apiGroup: rbac.authorization.k8s.io
    kind: Group
    name: cheesecake:users

```

so you can implement such rules/roles and RBs. Which should live in argocd/apps/values.yaml

Reply:
done